test/CMakeLists.txt

```diff
@@ -22,19 +23,44 @@ if(SPIRV_TOOLS_FOUND)
   endif()
 
   set(SPIRV_TOOLS_BINDIR "${SPIRV_TOOLS_PREFIX}/bin")
+elseif(SPIRV-Tools-tools_FOUND)
+  # we found SPIRV-Tools via cmake targets
+
+  if (TARGET spirv-as)
+    set(SPIRV_TOOLS_SPIRV_AS_FOUND True)
+  endif()
+
+  if (TARGET spirv-link)
+    set(SPIRV_TOOLS_SPIRV_LINK_FOUND True)
+  endif()
+
+  if (TARGET spirv-val)
+    set(SPIRV_TOOLS_SPIRV_VAL_FOUND True)
+  endif()
+
+  get_target_property(SPIRV_AS_PATH spirv-as IMPORTED_LOCATION_RELEASE)
+  if (NOT SPIRV_AS_PATH)
+	  get_target_property(SPIRV_AS_PATH spirv-as IMPORTED_LOCATION_DEBUG)
```

Comment:
nit: the indent here looks inconsistent.

Reply:
fixed